localization/english/wc_pol_faction_l_english.yml

```
@@ -0,0 +1,82 @@
+ found_horde_political_faction_confirm:0 "$found_alliance_political_faction_confirm$"
+ 
+ join_horde:0 "Join the Horde"
+ join_horde_desc:0 "$join_horde_desc$"
+ join_horde_tooltip:0 "$join_alliance_tooltip$"
+ join_horde_confirm:0 "$join_alliance_confirm$"
+ 
+ leave_horde:0 "Leave the Horde"
+ leave_horde_desc:0 "$leave_alliance_desc$"
+ leave_horde_tooltip:0 "$leave_alliance_tooltip$"
+ leave_horde_confirm:0 "$leave_alliance_confirm$"
+ 
+ THE_ALLIANCE:0 "the Alliance"
+ THE_HORDE:0 "the Horde"
+ 
+ WCPLF_100_TITLE:0 "[faction_leader.Custom('PolFaction')|U]"
+ WCPLF_100_DESC_beg:0 "Your messenger brings news that [faction_leader.GetFullNameNicknamed] announces that a new world power raises.\n\n"
+ WCPLF_100_DESC_alliance:0 "They call themselves the Alliance and welcome noble nations of Azeroth who are to weather the coming storm and preserve their cultures and peace. Driven by tradition, they respect deep commitments to abstract concepts like nobility, honor, faith, justice, and sacrifice. Striving to represent these high ideals, they contribute their technical, arcane, and spiritual wisdom toward the shared goal of preserving order on Azeroth and creating a peaceful and just world."
+ WCPLF_100_DESC_horde:0 "They call themselves the Horde and welcome disparate nations that struggle to survive in the hostile world that would see them destroyed. Driven by unity, they are bonded together as a family, comrades, or even uneasy allies. Focused, ferocious, and sometimes monstrous, the Horde values strength and honor and is ready to push back the ones threatening their ideals and shatter the status quo."
+ WCPLF_100_OPT_pos:0 "[faction_leader.Custom('PolFaction')|U] you say... should consider it."
+ WCPLF_100_OPT_neg:0 "[faction_leader.Custom('PolFaction')|U] you say... another enemy to fight with."
+ WCPLF_100_OPT_neu:0 "[faction_leader.Custom('PolFaction')|U] you say... keep an eye on them."
+ FOR_FACTION:0 "For [faction_leader.Custom('PolFaction')]!"
+ 
+ WCPLF_150_DESC:0 "My nation and I would like to join your side and become a part of [ROOT.Char.Custom('PolFaction')]."
+ 
+ WCPLF_155_DESC:0 "[recipient.Custom('PolFaction')|U] welcomes you! It would be an honor to fight side by side with you."
+ WCPLF_160_DESC:0 "After long consideration, I am afraid we have come to the conclusion that the [recipient.Custom('PolFaction')|U] must refuse your offer."
+ 
+ WCPLF_200_DESC:0 "It feels like [faction_leader.Custom('PolFaction')] has lost its leaders.\n\nAmong all who took the oath, your nation is the most respectful and strong. Your allies expect you to lead [faction_leader.Custom('PolFaction')] before it falls apart."
+ 
+ WCPLF_250_DESC:0 "[me.Custom('PolFaction')|U] has lost its leaders.\n\nFinding itself in a state of chaos, unable to find anyone to wear this burden, [me.Custom('PolFaction')] collapsed. The former members have stated that their previous agreements are no longer valid."
```

Slight change - not totally necessary but makes more sense
```suggestion
 WCPLF_250_DESC:0 "[me.Custom('PolFaction')|U] has lost its leaders.\n\nFinding itself in a state of chaos, unable to find anyone to shoulder this burden, [me.Custom('PolFaction')] collapsed. The former members have stated that their previous agreements are no longer valid."
```